ome/figure
# Shape export script

This adds shapes to the PDF and TIFF figure export.
NB: script will have to be manually updated each day - please ask if export not enabled.

To test:
 - Draw a bunch of shapes (Line, Arrow, Rectangle, Ellipse) well spread over a number of panels. Panels should have different zoom, pan & rotation. Also add scale bars and 'inner' & 'outer' labels.
 - Export as PDF and TIFF and check that all shapes appear exactly as they are in the web.
 - Shapes should be behind scale bar and other labels.
 - NB: shapes in PDF export are not cropped at the edge of the panel, but if the shape is entirely outside the panel (or centre of Ellipse is outside) then shape should not be added to page.
 - Test multi-page figures - shapes should be correctly positioned on each page.
